collector/nginx_plus.go

```diff
@@ -324,7 +410,7 @@ func (c *NginxPlusCollector) Collect(ch chan<- prometheus.Metric) {
 
 	for name, zone := range stats.ServerZones {
 		labelValues := []string{name}
-		varLabelValues := c.getServerZoneLabelValues(name)
+		varLabelValues := c.getStreamServerZoneLabelValues(name)
```

Comment:
This loop is for http server zones so this change should be reverted.
```suggestion
		varLabelValues := c.getServerZoneLabelValues(name)
```